terraform/testdata/provider-meta-unset/main.tf

```diff
@@ -0,0 +1,7 @@
+resource "test_instance" "foo" {
```

Comment:
These tests don't load any `provider_meta` blocks from modules, which I think is where they are intended to be used.

Reply:
Good point! Let me see if I can include that. I thought I'd simplify the testing by excluding it, but you're right that it's probably best if the tests all just cross module boundaries.